client/system/info_darwin.go

```diff
@@ -16,10 +18,15 @@ func GetInfo(ctx context.Context) *Info {
 	if err != nil {
 		fmt.Println("getInfo:", err)
 	}
+	out, err := exec.Command("sw_vers", "-productVersion").Output()
+	if err != nil {
+		fmt.Println("getInfo, macOS version error", err)
```

Comment:
let's use `release` for `version` in case of an error here.

Reply:
No, I don't know when sw_vers was introduced in macOS

Reply:
Do you want me to return the darwin version in case we have an error in getting the normal version

Reply:
yes

Reply:
regarding the command introduction, it seems to be there since the early X versions. The link https://www.manpagez.com/man/1/sw_vers/ lists it since 2009.